onnx/checker.cc

```diff
@@ -127,6 +140,31 @@ void check_tensor(const TensorProto& tensor, const CheckerContext& ctx) {
     for (const StringStringEntryProto& entry : tensor.external_data()) {
       if (entry.has_key() && entry.has_value() && entry.key() == "location") {
         has_location = true;
+#ifdef _WIN32
+        const fs::path windows_path(entry.value());
+        std::wstring relative_path = clean_relative_path(windows_path.wstring());
+        // Check that normalized relative path starts with "../" or "..\" on windows.
+        if (relative_path.rfind(L".." + w_k_preferred_path_separator, 0) == 0) {
```

Comment:
Why do you use rfind? Why search from end to start while the comment says you want to check if the string starts with something? 

And please replace == 0 with == std::wstring::npos.

Reply:
I followed the [original implementation](https://github.com/onnx/onnx/blob/8133ea176ca6aeb28a26a6f3e8e0687ed3c99a8e/onnx/checker.cc#L132), but I agreed it looks confusing. To prevent confusion, the code should reflect what the comment said. Thus, I changed it into:
```
if (relative_path.find(L".." + w_k_preferred_path_separator, 0) != std::string::npos) {
...
```